WordPress/Tests/NamingConventions/PrefixAllGlobalsUnitTest.php

```diff
@@ -32,7 +32,7 @@ public function getErrorList( $testFile = 'PrefixAllGlobalsUnitTest.1.inc' ) {
 		switch ( $testFile ) {
 			case 'PrefixAllGlobalsUnitTest.1.inc':
 				return array(
-					1   => 2, // 1 x error for blacklisted prefix passed.
+					1   => 8, // 1 x error for blacklisted prefix passed. 4 x error for short prefixes. 2 x no prefix.
```

Comment:
The description only totals 7 errors.

Reply:
Good catch. This was apparently introduced quite a while ago in this commit: https://github.com/WordPress/WordPress-Coding-Standards/commit/c23e34299d68b645168105f132f3f6013d94afa5#diff-4d42ce9e9b1e9d6a152b2fe03837da0eR401

But fixed it now.